Reading_files/vascular.ipynb

```diff
@@ -1,5 +1,16 @@
 {
```

Comment:
<p>1) You are importing glob, but I don't see where you are using it. Copuld you specify?</p><p>2) What's the difference between a sparse and dense matrix?</p><p>3) The importing packages section and the setting variable sections has to be separated. </p><p>4) Why are you using such a small image resolution? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Heart_Vascular/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>